boards/shields/tn0xxx/doc/index.rst

```diff
@@ -0,0 +1,76 @@
+.. _tn0xxx_shield:
```

Comment:
Please split it into two commits, one for the driver and bindings, and one to add the shield.

Reply:
done! thank you for the feedback